lib/options.js

```diff
@@ -47,7 +47,7 @@ const optionator = require("optionator");
  * @property {Object} [parserOptions] Specify parser options
  * @property {string[]} [plugin] Specify plugins
  * @property {string} [printConfig] Print the configuration for the given file
- * @property {boolean | undefined} reportUnusedDisableDirectives Adds reported errors for unused eslint-disable and eslint-enable directives
+ * @property {boolean | string | undefined} reportUnusedDisableDirectives Adds reported errors for unused eslint-disable and eslint-enable directives
```

Comment:
I'm guessing this is from an earlier version of the RFC. Now this option remains the same, but we should specify the new one.

Reply:
Fixed.